Components/TeamList.js

```diff
@@ -0,0 +1,78 @@
+import useSWR from "swr";
+import styled from "styled-components";
+
+const Header = styled.h1`
+  font-size: 5rem;
+  font-weight: bold;
+  margin-top: 1rem;
+  color: #000000; // #0d48a0 (Tory Blue / h2)
+  font-family: "roboto", sans-serif;
+  text-align: center;
+  letter-spacing: 3px;
+`;
+
+const StyledList = styled.ul`
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
+  grid-gap: 1.5rem;
+  list-style: none;
+  padding: 0;
+`;
+
+const StyledListItem = styled.li`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: #ffffff;
```

Comment:
I think the default background color is white so that could be removed

Reply:
True - removed it, ty!